pyInterface/bindings/decayAmplitude/massDependence_py.cc

```diff
@@ -452,5 +541,7 @@ void rpwa::py::exportMassDependence() {
 	bp::register_ptr_to_python<rpwa::piPiSWaveAuMorganPenningtonVesPtr>();
 	bp::register_ptr_to_python<rpwa::piPiSWaveAuMorganPenningtonKachaevPtr>();
 	bp::register_ptr_to_python<rpwa::rhoPrimeMassDepPtr>();
+	bp::register_ptr_to_python<rpwa::KpiSGLASSPtr>();
+	bp::register_ptr_to_python<rpwa::KpiSPalanoPenningtonPtr>();
```

Comment:
python interface for rpwa::binnedMassDependence is missing

Reply:
Done, two lines above.